app/models/application_record.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+
+  def esential
+    aplication = 'tranction'
```

Comment:
__[RuboCop]__

Lint/UselessAssignment: Useless assignment to variable - `aplication`.


<!-- Comment by [Sider](https://sider.review) -->